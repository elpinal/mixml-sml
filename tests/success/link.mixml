[unit
  link X = {} with
  {module u = [type :Type]}
]
