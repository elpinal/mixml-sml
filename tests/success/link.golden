[forall (!1@u : Type). {u : [= !1]}]+
